modules/renderer/map.js

```diff
@@ -217,8 +217,6 @@ export function Map(context) {
     function redraw(difference, extent) {
         if (!surface || !redrawEnabled) return;
 
-        clearTimeout(timeoutId);
```

Comment:
This overall looks really good to me, I can't wait to try it out.

I still think you'd need a `queueRedraw.cancel()` in here, because `redraw()` can be called from other places (e.g. data loaded from osm or other layers) - and you wouldn't want a stray debounced `redraw()` to sneak in after another one has already happened.